conversions.py

```diff
@@ -45,12 +45,24 @@ def generate_conversion_problem(start: Optional[str] = None, length: int = 3):
         end = random.choice(list(reactions[prev].values()))
         
         if c == 10000:
-            break
+        	break
 
         if length > 0 and end not in reactions:
             c += 1
             end = prev
             continue
         length -= 1
 
-    return start,end
\ No newline at end of file
+    return start,end
+
+def generate_problem():
+	from random import choice
+	start = choice(list(reactions))
+	solution = choice(list(reactions[start]))
+	solutions = [solution]
+	for i in range(2):
+		buffer = choice(list(reactions[choice(list(reactions))]))

```

Comment:
Wouldn't this raise an error if the reagent chosen wasn't in the reactions dict as a key?